modules/3d-tiles/wip/tileset-3d-full.md

```diff
@@ -71,15 +71,13 @@ The base path that non-absolute paths in tileset JSON file are relative to.
 
 ### maximumScreenSpaceError
 
-The maximum screen space error used to drive level of detail refinement. This value helps determine when a tile refines to its descendants, and therefore plays a major role in balancing performance with visual quality.
-
+Threshold that controls the level of detail of loaded tiles. A higher value means tile traversal stops early, displaying lower quality tiles (but much faster load times & less bandwidth used), because we're using a high "error tolerance". A lower value means lower tolerance for error, so traversal goes deeper in the tree and displays higher quality tiles. 
```

Comment:
nice! this updated documentation is much clearer and more helpful. thank you for adding it!